src/me.rs

```diff
@@ -1028,28 +1027,28 @@ pub mod test {
   // Regression and validation test for SAD computation
   fn get_sad_same_inner<T: Pixel>() {
     let blocks: Vec<(BlockSize, u32)> = vec![
-      (BLOCK_4X4, 1912),
-      (BLOCK_4X8, 4296),
-      (BLOCK_8X4, 3496),
-      (BLOCK_8X8, 7824),
-      (BLOCK_8X16, 16592),
-      (BLOCK_16X8, 14416),
-      (BLOCK_16X16, 31136),
-      (BLOCK_16X32, 60064),
-      (BLOCK_32X16, 59552),
-      (BLOCK_32X32, 120128),
-      (BLOCK_32X64, 186688),
-      (BLOCK_64X32, 250176),
-      (BLOCK_64X64, 438912),
-      (BLOCK_64X128, 654272),
-      (BLOCK_128X64, 1016768),
-      (BLOCK_128X128, 1689792),
-      (BLOCK_4X16, 8680),
-      (BLOCK_16X4, 6664),
-      (BLOCK_8X32, 31056),
-      (BLOCK_32X8, 27600),
-      (BLOCK_16X64, 93344),
-      (BLOCK_64X16, 116384),
+        (BLOCK_4X4, 1912),
+        (BLOCK_8X4, 3496),
+        (BLOCK_4X8, 4296),
+        (BLOCK_8X8, 7824),
+        (BLOCK_16X8, 14416),
+        (BLOCK_8X16, 16592),
+        (BLOCK_16X16, 31136),
+        (BLOCK_32X16, 59552),
+        (BLOCK_16X32, 60064),
+        (BLOCK_32X32, 120128),
+        (BLOCK_64X32, 250176),
+        (BLOCK_32X64, 186688),
+        (BLOCK_64X64, 438912),
+        (BLOCK_128X64, 1016768),
+        (BLOCK_64X128, 654272),
+        (BLOCK_128X128, 1689792),
+        (BLOCK_16X4, 6664),
+        (BLOCK_4X16, 8680),
+        (BLOCK_32X8, 27600),
+        (BLOCK_8X32, 31056),
+        (BLOCK_64X16, 116384),
+        (BLOCK_16X64, 93344),
```

Comment:
You can kept this exactly like it was. This is an unnecessary diff (and the blocks are not in the same order as in the definition of `BlockSize`).